ludwig/schema/metadata/configs/encoders.yaml

```diff
@@ -1,3 +1,10 @@
+BaseEncoder:
+    skip:
+        internal_only: true
+        other_information: Internal Only
+        related_parameters:
```

Comment:
I would actually just leave `related_parameters` out since by default it will fill it with `None` which will be easier to deal with in the future!